tests/test_logging.py

```diff
@@ -8,91 +7,89 @@
 import opsdroid.logging as opsdroid
 from opsdroid.cli.start import configure_lang
 
+configure_lang({})
 
-class TestLogging(unittest.TestCase):
-    """Test the logging module."""
 
-    def setUp(self):
-        self._tmp_dir = os.path.join(tempfile.gettempdir(), "opsdroid_tests")
-        with contextlib.suppress(FileExistsError):
-            os.makedirs(self._tmp_dir, mode=0o777)
-        configure_lang({})
+@pytest.fixture(scope="module")
```

Comment:
Sure. You don't have this file yet, am I right? I can't seem to find it. 

btw pytest's way is using a file called conftest.py, see https://docs.pytest.org/en/stable/fixture.html#conftest-py-sharing-fixture-functions, so maybe that's what I would do, if it is ok for you.

Reply:
We have a fixtures file already, you can find it here: https://github.com/opsdroid/opsdroid/blob/master/opsdroid/testing/fixtures.py 

If you can't see it you might need to update your fork, if you need help doing that let me know

Reply:
Got it! I'll push in a minute.

Reply:
So moving the function to fixtures made me import it, but then it does not behave like a pytest fixture, so I can't include it in the function parameter list, but use it in the function from the outer scope. With conftest.py it might have been better.

Reply:
Ahh and it fails because of this.
But if I leave the _tmp_dir fixture in the parameter list of the function, then black won't let me commit, with:
```
opsdroid/tests/test_logging.py:8:1: F401 'opsdroid.testing.fixtures._tmp_dir' imported but unused
```
And several others, like:
```
opsdroid/tests/test_logging.py:33:33: F811 redefinition of unused '_tmp_dir' from line 8
```

Reply:
How can I make black silent? :shushing_face: 
Or do you have a better idea?